### ISSUE TITLE
Insert method doesn't work when element is select tag

### ISSUE THREAD

**Issue by Wojdav** [AngleSharp/AngleSharp#449]
I have sample code, where x is a `select` element. MWE:

``` cs
x.Insert(AdjacentPosition.AfterEnd, "<p class=\"cls\">Text</p>")
```

The result should be `<select></select><p class="cls">Text</p>`, but it's `<select></select>Text`.

If I change the select element to something else it's working.

**Comment by FlorianRappl** [comment]
Yep, I can confirm this bug. Will be fixed in the upcoming release! Thanks for the report 👍 .

**Comment by FlorianRappl** [comment]
Fixed, see PR #450.
